vignettes/hys_law.Rmd

```diff
@@ -0,0 +1,156 @@
+---
+title: "Hy's Law Implementation Guide"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Hy's Law Implementation Guide}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+
+library(admiral)
+library(admiraldev, warn.conflicts = FALSE)
+library(dplyr, warn.conflicts = FALSE)
+library(tidyr, warn.conflicts = FALSE)
+library(purrr)
+library(admiral.test)
+library(lubridate)
+library(stringr)
+```
+
+# Context
+
+During the drug development process, clinical trials are often required to assess for the potential that the experimental drug can cause severe liver injury, known as a drug induced liver injury (DILI). There are multiple criteria that need to be evaluated to determine and classify a DILI "event". A common rule of thumb for this is, known as Hy's Law, is usually comprised of three parts:
+
+- Elevated alanine aminotransaminase (ALT) or aspartate aminotransaminase (AST) by 3-fold or greater of the upper limit of normal.
+- Elevated serum total bilirubin (BILI) by 2-fold or greater within a window of time, ~14 days after the elevated AST/ALT event.
```

Comment:
Instead of fold can we say 3 times or 2 times.  Or is the fold common lab talk??

Reply:
Feels like we can swap to "times" FDA document seems to reference both "fold" and "x" synonymously. 
![image](https://user-images.githubusercontent.com/43018762/219444974-38cf868e-2eea-4769-aca2-d82784b55cbe.png)